ISSUE_TEMPLATE/next.md

```diff
@@ -111,6 +111,21 @@ curl -H 'Accept: application/json' 'https://updates.coreos.fedoraproject.org/v1/
 
 NOTE: In the future, most of these steps will be automated.
 
+## Update the release notes
+
+NOTE: You can do this simultaneously for all releases that are happening at the same time.
+
+- For each [issue that is pending a next release in the tracker](https://github.com/coreos/fedora-coreos-tracker/issues?q=is%3Aissue+label%3Astatus%2Fpending-next-release):
+  - [ ] Add an entry to the release notes file: `release-notes/next.yml`
+  - [ ] Add a message to the issue saying that the fix as been released:
+    ```
+    The fix for this went into the next stream release X.Y.Z. Please try out the new release and report issues.
+    ```
+  - [ ] Remove the `status/pending-next-release` label from the issue.
```

Comment:
If applicable the `status/pending-testing-release` label should be added to the issue.